services-directions-models/src/main/java/com/mapbox/api/directions/v5/utils/FormatUtils.java

```diff
@@ -169,4 +169,27 @@ public static String formatPointsList(@Nullable List<Point> points) {
     }
     return join(";", coordinatesToJoin);
   }
+
+  /**
+   * Converts list of Z-levels to a string ready for API consumption.
+   *
+   * @param zLevels a list representing Z-levels
+   * @return a formatted string with semicolon separated integers
+   */
+  @Nullable
+  public static String formatZLevels(@Nullable List<Integer> zLevels) {
```

Comment:
```suggestion
  public static String formatIntegers(@Nullable List<Integer> integers) {
```
this can be generic, there's not logic specific to z-levels in this function